### PR TITLE
Require ext-json in composer

### PR DESCRIPTION
**Pull request by thePanz**

### CODE REVIEW

**Comment by ruflin** [comment]
Could you share some background on why we need this? Was it missing?

Quick note on branches: I would prefer if you could branches in your fork and open PR's from there so. Unfortunately I can't do that too as it's in my repo :-(

**Comment by thePanz** [comment]
@ruflin the requirement is sort of "implicit" from the `elasticsearch/elasticaserach` dependency.

This PR makes it explicit, as we use `json_decode()` functions, too.

File: composer.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     ],
     "require": {
         "php": "^7.1",
+        "ext-json": "*",
```
Comment (ruflin) [inline composer.json]: Could we use a more specific version here?

Comment (thePanz) [reply composer.json]: PHP extensions have no version, either they're enabled or not :)

**Pull request merged**